app/bundles/LeadBundle/Tests/Entity/LeadFieldRepositoryFunctionalTest.php

```diff
@@ -154,13 +154,13 @@ public function testCompareValueInOperatorWithMultiselectField(): void
         );
 
         $fieldModel = self::getContainer()->get(FieldModel::class);
-        \assert($fieldModel instanceof FieldModel);
+        \assert($fieldModel instanceof FieldModel); // @phpstan-ignore-line
```

Comment:
Why is this causing an issue for PHPSTAN? Can you add the reason for it after the ignore so it's clear? There is one more such occurrence in this file